crowbar_framework/app/models/node_object.rb

```diff
@@ -170,6 +170,10 @@ def self.all
     self.find nil
   end
 
+  def self.admin_node
+    all.detect(&:admin?)
```

Comment:
For the record, we have some place in the code that was using this: `NodeObject.find("role:crowbar").find { |n| n.admin? }`

We should probably make this consistent. I do like the extra check on the crowbar role, btw.